src/model/point-model.js

```diff
@@ -11,8 +11,8 @@ export default class PointModel {
 
   init() {
     this.#offers = offersData;
-    this.#points = pointsMock;
-    this.#destinations = destinationsMock;
+    this.#points = mockPoint;
```

Comment:
лучше тогда такой нейминг
```suggestion
    this.#points = mockedPoint;
```